apple/xcode/osx/www/cordova_plugins.js

```diff
@@ -0,0 +1,44 @@
+cordova.define('cordova/plugin_list', function(require, exports, module) {
```

Comment:
Could you add a comment explaining why this was added and what needs to be done to clean up?

Reply:
Done in afd52c8.